app/views/packages/show.html.erb

```diff
@@ -104,59 +105,51 @@
               </div>
             <% end %>
           </div>
-          <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
-            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
-            <span class="sr-only">Previous</span>
-          </a>
-          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
-            <span class="carousel-control-next-icon" aria-hidden="true"></span>
-            <span class="sr-only">Next</span>
-          </a>
-        </div>
-      </div>
+
+
+                <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
+                  <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+                  <span class="sr-only">Previous</span>
+                </a>
+                <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+                  <span class="carousel-control-next-icon" aria-hidden="true"></span>
+                  <span class="sr-only">Next</span>
+                </a>
+
+    </div>
   </div>
-</div>
 
 
 <script>
   const unavailableTimes = JSON.parse(document.querySelector('.time-slots').dataset.unavailable)
   console.log(unavailableTimes)
-
   const updateForm = (event) => {
     const slot = event.currentTarget.innerHTML
     const startTime = slot.split("-")[0]
     const endTime = slot.split("-")[1]
     document.querySelector('#booking_start_time').value = startTime
     document.querySelector('#booking_end_time').value = endTime
   }
-
   const changeColor = (event) => {
     // Set backgrond of all other slots to white, so it is clear only one can be selected
     document.querySelectorAll(".time-slot").forEach( (slot) => {
       if (slot.classList.contains('time-slot-selected')) {
         slot.classList.remove("time-slot-selected");
       };
     })
-
-
     const slot = event.currentTarget;
     slot.classList.toggle("time-slot-selected");
   }
-
   const timeSlots = document.querySelectorAll(".time-slot");
   timeSlots.forEach((slot) => {
     if (unavailableTimes.includes(parseInt(slot.innerHTML.split("-")[0], 10))) {
       slot.classList.add("unavailable-slot");
     } else {
       slot.classList.add("available-slot");
     }
-
     slot.addEventListener('click', updateForm);
     if (slot.classList.contains('available-slot')) {
       slot.addEventListener('click', changeColor);
     };
-
   })
 </script>
```

Comment:
You got it working? Looks good! But do try to have everything lined up nicely, like the divs of 'carousel-item' and 'photographerModal'. should start on the same column. I've found that (in Sublime) Edit > Line > Reindent can definitely help. Only it doesnt indent Ruby iterations, which still have to be done. This also made me notice you are probably missing a final closing div for the main container?

Reply:
I'll count the divs again